macros/src/actix.rs

```diff
@@ -659,6 +659,44 @@ pub fn emit_v2_errors_overlay(attrs: TokenStream, input: TokenStream) -> TokenSt
     gen.into()
 }
 
+fn extract_rename(attrs: &[Attribute]) -> Option<String> {
```

Comment:
We already have an `extract_openapi_attrs` - could we make use of it here?
Btw, while you're at it could you replace the string "openapi" in `extract_openapi_attrs` with `SCHEMA_MACRO_ATTR`?